browser/css/menubar.css

```diff
@@ -16,7 +16,7 @@
 
 #document-header {
 	position: relative;
-	background: #ffffff;
+	background: transparent;
```

Comment:
I have created a PR with a commit just for the colours, maybe we can divide further this PR into several commits, I will help you with that.

Reply:
https://github.com/CollaboraOnline/online/pull/3906